e2e/rnTestUtilities.js

```diff
@@ -80,12 +80,111 @@ const assertAutotrackHierarchy = async (
   }
 };
 
-waitForEventsToFlush = async () => {
-  // Heap for iOS and Android flushes events every 15 seconds. Wait 16 seconds to ensure that
-  // all events are flushed to redis before asserting.
-  // :TODO:(jmtaber129): Make this wait shorter if/when we expose setting the
-  // flush frequency to the RN bridge.
-  await new Promise(resolve => setTimeout(resolve, 16000));
+const assertAndroidNavigationEvent = async (expectedPath, expectedType) => {
+  let props = {
```

Comment:
Style nit: could get away with a `const` by extracting the initial `{ path: ... }` object to another `const commonProps` or similar.

Reply:
In more detail, here's the sort of change I'd suggest, but it's mostly a style preference. I don't have a strong opinion here.

```ts
const commonProps = {
  path: {
    string: expectedPath,
  }
};

const props = expectedType
	? commonProps
	: { 
      ...commonProps, 
      type: {
        string: expectedType,
      }
    };
```

Could also just go for a little duplication to reduce cognitive load of what's goin on in that spread syntax.

```ts
const props = expectedType
  ? {
      path: {
        string: expectedPath
      }
    }
  : {
      path: {
        string: expectedPath
      },
      type: {
        string: expectedType
      }
    };
```

Reply:
done!